### DISCUSSION TITLE
Enhancements

### DISCUSSION THREAD

**Discussion by immiProgrammer** [jln13x/ui.jln.dev#14] [Ideas]
it is possible to change whole site theme according the current theme selected/generated
this case its looks not greate
![image](https://github.com/jln13x/ui.jln.dev/assets/101139501/7c0b1488-06e6-4735-a5a8-204c3e53c49f)

**Comment by immiProgrammer** [comment]
when i paste color value in color picker the error message is showing for invalid theme paste
![image](https://github.com/jln13x/ui.jln.dev/assets/101139501/a6330ef9-c650-4404-b791-c13419aa617a)

**Reply by jln13x** [reply]
but its still pasting the color right?

**Reply by immiProgrammer** [reply]
Yes

**Comment by immiProgrammer** [comment]
to fix pasting u can get first focus element then check if not color dialog or input/text then ignore paste invalid theme error

**Reply by jln13x** [reply]
wanna submit a pr for that? :)

**Reply by immiProgrammer** [reply]
> wanna submit a pr for that? :) 

Ok i m try

**Comment by programming-with-ia** [comment]
@jln13x, have you seen this? 👀 What do you think?
https://ui.shadcn.com/docs/tailwind-v4#whats-coming-next
> Migrating colors to OKLCH.
